websites_schema.yml

```diff
@@ -18,24 +18,26 @@ mapping:
           "img":
             type: str
             required: yes
+            pattern: /\.png$/i
           "tfa":
             type: bool
             required: yes
+            pattern: /(true|false)/
```

Comment:
i'm not sure, is this redundant? `bool` should only be able to be set to `true` or `false`. could you override this with a `nil` or something else?

Reply:
the validate command in the verify.rb will appreciate this more along with allowing for better handling if you add in more verifications written out manually for those fields.

Reply:
sorry, i'm just especially dense this morning and i'm not understanding.

could you expand on your answer? still don't understand how this isn't redundent

Reply:
looking over things, I cannot find where it was that I was having trouble. Maybe you can ignore this if you think its redundant.